Frontend/tailored-brand-poc/src/app/invoice-detail/invoice-detail.component.ts

```diff
@@ -137,7 +139,7 @@ export class InvoiceDetailComponent implements OnInit {
             if (res.status == 200) {
               let len = res.ids.length
               for (let i = 0; i < len; i++) {
-                this.createNewFormControl(res.ids[i], res.quantity[i],res.quantityPending[i])
+                this.createNewFormControl(res.ids[i], res.quantity[i],0)
```

Comment:
start using named params in functions, use ES6 restructuring. It will help any one understand what these params are.

Reply:
Ok , i will do that.